### PR TITLE
Remove bazel from circleci integration test

### PR DESCRIPTION
**Pull request by rshriram**
```release-note
NONE
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -22,55 +27,14 @@ jobs:
...
-      - run: cd pilot; mkdir /home/circleci/logs; make e2etest HUB=$HUB TAG=$TAG TESTOPTS="-mixer=false -errorlogsdir=/home/circleci/logs"
+            - dep-cache-{{ checksum "Gopkg.lock" }}
+      - run: bin/purego-build-pilot -hub $HUB -tag $TAG -build-only
+      - run: mkdir /home/circleci/logs
+      - run: go build -i ./pilot/test/integration
+      - run: ./integration --logtostderr -hub $HUB -tag $TAG -mixer=false -errorlogsdir=/home/circleci/logs 
       - store_artifacts:
           path: /home/circleci/logs
 
-  unittest:
```
Comment (ijsnellf) [inline .circleci/config.yml]: are you removing `unittest` or did you just remove it temporarily to make experimentation faster?

**Comment by ijsnellf** [comment]
is this wip?

File: bin/purego-build-pilot [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+go build -i ./pilot/cmd/pilot-agent
+go build -i ./pilot/cmd/pilot-discovery
+go build -i ./pilot/cmd/sidecar-initializer
+go build -i ./pilot/test/server
+go build -i ./pilot/test/client
+go build -i ./pilot/test/eurekamirror
+
+# Collect artifacts for pushing
+cp -f  client pilot/docker/client
```
Comment (kyessenov) [inline bin/purego-build-pilot]: instead of copying, do `go build -o client`, etc.

File: .circleci/config.yml (hunk 1 above)
Comment (rshriram) [reply .circleci/config.yml]: temporary.. fixed

**Pull request merged**